File: eBarberShop/eBarberShop.Model/Usluga.cs
namespace eBarberShop.Model
{
    public class Usluga
    {
        public int UslugaId { get; set; }
        public string Naziv { get; set; }
        public string? Opis { get; set; }
        public decimal Cijena { get; set; }
        public int? Trajanje { get; set; }
        public byte[]? Slika { get; set; }

        public virtual ICollection<SlikeUsluge> SlikeUsluge { get; set; } = new List<SlikeUsluge>();
    }
}

File: eBarberShop/UI/ebarbershop_mobile/lib/models/usluga/usluga.dart
import 'package:json_annotation/json_annotation.dart';

part 'usluga.g.dart';

@JsonSerializable()
class Usluga {
  int uslugaId;
  String naziv;
  String? opis;
  double cijena;
  int trajanje;
  String? slika;

  Usluga(this.uslugaId, this.naziv, this.opis, this.cijena, this.trajanje, this.slika);

  factory Usluga.fromJson(Map<String, dynamic> json) =>
      _$UslugaFromJson(json);

  Map<String, dynamic> toJson() => _$UslugaToJson(this);
}